sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -248,6 +242,15 @@ def _get_available_query_metrics_columns(self, cursor, all_expected_columns):
         self.log.debug("found available sys.dm_exec_query_stats columns: %s", available_columns)
         return available_columns
 
+    # only works for non-encrypted procedures, but if
+    # text is encrypted, we will use the plan_handle as the RL
+    # key to decide when to lookup the plan
+    @staticmethod
+    def _get_is_proc(row):
+        if not row['is_encrypted']:
+            return "CREATE PROCEDURE" in row['text']
```

Comment:
This should be one function with the above to check if a proc, not two separate functions. The extra check on `is_encrypted` can happen outside of the function.

Reply:
we can actually just check if `'text' in row`